src/provider/dynt.rs

```diff
@@ -40,6 +40,13 @@ impl Tpm for BoxedDynTpm {
         self.0.root_storage_key_create(auth_value)
     }
 
+    fn machine_key_create(
+        &mut self,
+        auth_value: &AuthValue,
+    ) -> Result<LoadableStorageKey, TpmError> {
+        self.0.root_storage_key_create(auth_value)
```

Comment:
@Firstyear perhaps we want to return a deprecation error here, instead of calling the correct function?

Reply:
and ditto to the others